src/thenCatchP.js

```diff
@@ -0,0 +1,29 @@
+import { invoker } from 'ramda';
+
+/**
+ * Composable shortcut for `Promise.then` that allows for success and failure call backs.
+ * The thenCatchP function returns a Promise. It takes three arguments: a callback function for the success of the Promise,
+ * a callback function for the failure of the Promise, and the promise instance itself.
+ *
+ * @func thenP
```

Comment:
```suggestion
 * @func thenCatchP
```

Reply:
😏  Guess you can tell I stole the description block from `thenP`.

Reply:
yup! I do the same ;]